client/package.json

```diff
@@ -20,6 +20,7 @@
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "typescript-eslint": "0.0.1-alpha.0",
```

Comment:
Not sure why the client side was changed. Is it possible you installed eslint globally or in the root?